diff --git a/lua/plugins/configs/nvterm.lua b/lua/plugins/configs/nvterm.lua
deleted file mode 100644
index 720b6d2..0000000
--- a/lua/plugins/configs/nvterm.lua
+++ /dev/null
@@ -1,43 +0,0 @@
-local present, nvterm = pcall(require, "nvterm")
-
-if not present then
-   return
-end
-
-local options = {
-   terminals = {
-      list = {},
-      type_opts = {
-         float = {
-            relative = "editor",
-            row = 0.3,
-            col = 0.25,
-            width = 0.5,
-            height = 0.4,
-            border = "single",
-         },
-         horizontal = { location = "rightbelow", split_ratio = 0.3 },
-         vertical = { location = "rightbelow", split_ratio = 0.5 },
-      },
-   },
-   behavior = {
-      close_on_exit = true,
-      auto_insert = true,
-   },
-   mappings = {
-      toggle = {
-         float = "<A-i>",
-         horizontal = "<A-h>",
-         vertical = "<A-v>",
-      },
-      new = {
-         horizontal = "<leader>h",
-         vertical = "<leader>v",
-      },
-   },
-   enable_new_mappings = true,
-}
-
-options = require("core.utils").load_override(options, "NvChad/nvterm")
-
-nvterm.setup(options)
diff --git a/lua/plugins/configs/others.lua b/lua/plugins/configs/others.lua
index 55b2561..ba788b1 100644
--- a/lua/plugins/configs/others.lua
+++ b/lua/plugins/configs/others.lua
@@ -18,24 +18,6 @@ M.autopairs = function()
    cmp.event:on("confirm_done", cmp_autopairs.on_confirm_done())
 end
 
-M.better_escape = function()
-   local present, escape = pcall(require, "better_escape")
-
-   if not present then
-      return
-   end
-
-   local options = {
-      mapping = { "jk" }, -- a table with mappings to use
-      timeout = vim.o.timeoutlen,
-      clear_empty_lines = false, -- clear line after escaping if there is only whitespace
-      keys = "<Esc>",
-   }
-
-   options = require("core.utils").load_override(options, "max397574/better-escape.nvim")
-   escape.setup(options)
-end
-
 M.blankline = function()
    local present, blankline = pcall(require, "indent_blankline")
 
diff --git a/lua/plugins/init.lua b/lua/plugins/init.lua
index 727b99c..5ab6061 100644
--- a/lua/plugins/init.lua
+++ b/lua/plugins/init.lua
@@ -26,12 +26,6 @@ local plugins = {
       end,
    },
 
-   ["NvChad/nvterm"] = {
-      config = function()
-         require "plugins.configs.nvterm"
-      end,
-   },
-
    ["kyazdani42/nvim-web-devicons"] = {
       after = "base46",
       config = function()
@@ -127,13 +121,6 @@ local plugins = {
       end,
    },
 
-   ["max397574/better-escape.nvim"] = {
-      event = "InsertCharPre",
-      config = function()
-         require("plugins.configs.others").better_escape()
-      end,
-   },
-
    -- load luasnips + cmp related in insert mode only
 
    ["rafamadriz/friendly-snippets"] = {
